cli/lib/tasks/cache.js

```diff
@@ -15,15 +27,51 @@ const clear = () => {
 
 const list = () => {
   return getCachedVersions()
-  .then((versions) => {
-    logger.log(versions.join(', '))
+  .then((binaries) => {
+    const table = new Table({
+      head: [a('version'), a('last used')],
+    })
+
+    binaries.forEach((x) => {
+      const versionString = g(x.version)
+      const lastUsed = x.accessed ? u(x.accessed) : 'unknown'
```

Comment:
Again, what is `x` here? This variable name makes this method harder to read. Suggest calling it `binary` if it is a singular of `binaries. Clarity > brevity.